pico/ff_implementation/OperatorsFFNodes/InOut/ReadFromStdInFFNode.hpp

```diff
@@ -52,6 +52,32 @@ class ReadFromStdInFFNode: public base_filter {
 		assert(false);
 	}
 
+
+
+	void extract_from_buffer(char* buffer, int n, std::string &tail, Microbatch<TokenType>* &mb) {
+		tail.append(buffer, n);
```

Comment:
It seems this mechanism on `tail` can be very inefficient. In case of long lines (i.e., longer than `CHUNK_SIZE`), the partial line is scanned multiple times from the beginning.

Let us figure out something better, I will also think about that.

Reply:
We can apply the solution used for ReadFromFileFFNode_seq. You can find it in ReadFromFileFFNode.hpp

Reply:
I agree, let us get rid of the additional buffering, it is probably a leftover from some dismissed approach.

Reply:
Done